tests/test_project.py

```diff
@@ -4,18 +4,19 @@
 from sys import platform
 from typing import Sequence
 
+from pyprojroot.here import here
```

Comment:
Is this dependency really necessary? I guess it's convenient, but the package seems to be dying already (a year since last update). Also it feels a bit like dependency bloat in the style of JavaScript "left align" packages. I could get over the latter if this was an actively maintained and broadly used package...

Reply:
True, I do not like it either, it is a temporary fix, as `pytest-copie` plugin that we use does not seem to support 'session-scope', so we had to improvise.